Update try it online in the documentation. (#967)
jupyter-widgets/ipyleaflet: docs/source/index.rst

Try it online
-------------

> Directly from this page, you can make a try with ipyleaflet with clicking on `Try it on RetroLab <https://ipyleaflet.readthedocs.io/en/latest/lite/retro>`__ or

Reviewer: ```suggestion
You can give ipyleaflet a try in this documentation by clicking on `Try it on RetroLab <https://ipyleaflet.readthedocs.io/en/latest/lite/retro>`__ or
```
